.github/workflows/java_mirror.yml

```diff
@@ -0,0 +1,25 @@
+name: mirror
```

Comment:
I think this part is obsolete. It used to mirror to internal repo in Gojek, but that repo is on 0.9